src/snowflake/cli/plugins/snowpark/package_utils.py

```diff
@@ -231,9 +227,20 @@ def pip_wheel(
     return process.returncode
 
 
-def get_package_dependencies(
-    requirements_file: SecurePath, downloads_dir: Path
-) -> List[RequirementWithWheelAndDeps]:
+@dataclasses.dataclass
+class SplitDownloadedDependenciesResult:
+    snowflake_dependencies: List[Requirement] = dataclasses.field(default_factory=list)
```

Comment:
```suggestion
    avilable_in_snowflake_dependencies: List[Requirement] = dataclasses.field(default_factory=list)
```

Just to avoid confusion that something is "dependecy for snowflake". WDYT?